platforms/ios/example/Wysiwyg/ContentView.swift

```diff
@@ -37,11 +38,25 @@ struct ContentView: View {
             viewModel.clearContent()
         }
         .disabled(viewModel.isContentEmpty)
-        .accessibilityIdentifier("WysiwygSendButton")
+        .accessibilityIdentifier(.sendButton)
         if let sentMessage = sentMessage {
-            Text("Content: \(sentMessage.plainText)\nHTML: \(sentMessage.html)")
-                .accessibilityIdentifier("WysiwygContentText")
+            VStack {
+                HStack {
+                    Text("Content:")
+                    Text(sentMessage.plainText)
+                        .accessibilityIdentifier(.contentText)
+                }
+                HStack {
+                    Text("HTML:")
+                    Text(sentMessage.html)
+                        .accessibilityIdentifier(.htmlContentText)
+                }
+            }
         }
         Spacer()
     }
 }
+
+private extension View {
```

Comment:
Is this supposed to be here even though it's empty? (Forgive my ignorance)

Reply:
Nope, I missed it 🙈